src/node/internal/internal_zlib.ts

```diff
@@ -3,13 +3,13 @@
 //     https://opensource.org/licenses/Apache-2.0
 // Copyright Joyent and Node contributors. All rights reserved. MIT license.
 
-import { default as zlibUtil, type ZlibOptions } from 'node-internal:zlib';
+import { default as zlibUtil, ZlibOptions } from 'node-internal:zlib';
```

Comment:
This needs to be reverted 

```suggestion
import { default as zlibUtil, type ZlibOptions } from 'node-internal:zlib';
```